client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java

```diff
@@ -145,4 +145,24 @@ public static void applyDynamicClientConf(JobConf jobConf, Map<String, String> c
       }
     }
   }
+
+  public static int getInt(JobConf priorityConf, JobConf normalConf, String key, int defaultValue) {
```

Comment:
The name of priorityConf and normalConf are strange.

Reply:
`FirstConf SecondConf` may be better?

Reply:
conf and defaultConf?

Reply:
Not good. `rssJobConf and jobConf` ?

Reply:
rssJobConf and mrJobConf ？

Reply:
ok.